src/components/cc-domain-management/cc-domain-management.smart.md

```diff
@@ -0,0 +1,74 @@
+---
+kind: '🛠️ Domains/<cc-domain-management>'
+title: '💡 Smart'
+---
+
+# 💡 Smart `<cc-domain-management>`
+
+## ℹ️ Details
+
+<!-- TODO: -->
+<table>
+  <tr><td><strong>Component    </strong> <td><a href="https://www.clever-cloud.com/doc/clever-components/?path=/docs/🛠-organisation-cc-orga-member-list--default-story"><code>&lt;cc-orga-member-list></code></a>
```

Comment:
TODO @florian-sanders-cc: once we agree on the component name, edit this link and remove the todo comment.